spec/spec_helper.rb

```diff
@@ -2,9 +2,6 @@
 require 'simplecov-rcov'
 SimpleCov.formatter = SimpleCov::Formatter::RcovFormatter
 SimpleCov.minimum_coverage 100
-SimpleCov.start do
-  add_filter '/.bundle/'
-end
```

Comment:
If you really need to add `.bundle`, see further down in the file. I suspect that `/gem/` might cover it - if not, change it.